.github/workflows/scripts/evaluate-jobs-conclusions.js

```diff
@@ -0,0 +1,91 @@
+/* eslint-disable no-console */
+const { REPOSITORY, RUN_ID, GITHUB_TOKEN, TEST_MODE } = process.env;
+const IGNORED_JOBS = [
+	'Evaluate Project Job Statuses',
+	'Report e2e tests results',
+	'Report API tests results',
+];
+
+const isJobRequired = ( job ) => {
+	return (
+		! job.name.endsWith( '(optional)' ) &&
+		! IGNORED_JOBS.includes( job.name )
```

Comment:
GitHub API will return all the jobs from this workflow and we need to filter the ones that are not needed. These jobs could have already been completed and failed, and we don't want them to interfere with the result.